audit-connector/src/main/java/com/sequenceiq/cloudbreak/audit/model/AuditEventName.java

```diff
@@ -1,5 +1,14 @@
 package com.sequenceiq.cloudbreak.audit.model;
 
 public enum AuditEventName {
-    DATAHUB_CLUSTER_CREATION
+    DATAHUB_CLUSTER_CREATION,
```

Comment:
unfortunately, the naming will change, therefore a parameter will be needed.
Reference: https://github.infra.cloudera.com/thunderhead/thunderhead/blob/master/services/audit/event-names.md
Bill will update the sheet with the names (has not started it yet...) I will let you know, but you can prepare for it.

Reply:
The updated names are in the "CDP Control Plane Audit Events" sheet (contact me for URL please if you don't have it)